Sources/Controllers/SpotifyController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  SpotifyController.swift
+//  ControlKit
+//
+
+import SpotifyiOS
+import SwiftUI
+
+/// Wrapper for ``SPTAppRemote``.
+public final class SpotifyController: NSObject, ObservableObject {
+    
+    lazy private var remote: SPTAppRemote = {
+        let remote = SPTAppRemote(
+            configuration: SPTConfiguration(
+                clientID: SpotifyConfig.clientID,
+                redirectURL: URL(string: SpotifyConfig.redirectURL)!
+            ),
+            logLevel: .debug
```

Comment:
What is your opinion on switching the severity of log messages to `error` when there is no DEBUG flag? 👨‍🔧

Reply:
Sounds good, I'm just wondering how to access the `DEBUG` flag here 🤔

Would you suggest we declare a var somewhere and check that when setting the `logLevel` parameter?

```swift
var isDebug = false
#if DEBUG
isDebug = true
#endif
```

Reply:
I've ended up adding the log level as a property of `SpotifyConfig` with the default level as `info`. This way the user of the package can set it themselves 👷‍♂️

Reply:
I've ended up adding the log level as a property of `SpotifyConfig` with the default level as `info`. This way the user of the package can set it themselves 👷‍♂️